sysml/src/examples/Geometry Examples/VehicleGeometryAndCoordinateFrames.sysml

```diff
@@ -42,45 +86,46 @@ package VehicleGeometryAndCoordinateFrames {
         part chassis : Chassis[1] :> componentItems {
 			attribute :>> coordinateFrame {
```

Comment:
Better to use `nestedCoordinateFrame`, as `coordinateFrame` is redefined for `componentItems`. This is a diamond inheritance issue and it may not otherwise pass validation in the future.

Reply:
I see why it has to be like this with the current SpatialItem and CompoundSpatialItem setup, and I think we should keep it like this for now. But it is a quite complicated structure that most end-users will find extremely difficult to grok and use. 

So, I feel we need to raise a new issue to make a concerted effort to simplify how to model composite structure with spatial extent in SysML in a way that is similar / analogous to a CAD / 3D modeling assembly tree with leaf parts, without the necessity to change item/part type and coordinateFrame attribute when a leaf of the tree is changed into a (sub)assembly with more detailed lower level parts. This is needed to evolve a physical architecture e.g. from conceptual through preliminary to detailed design or to 'defeature' a 3D model in the other direction from detailed to approximate / lower fidelity. Or indeed to enable a single model supporting different levels of detail / fidelity, perhaps using variants.

Reply:
Agreed that there should be a new issue, probably in Track 4, if we don't already have an issue to cover this there.